3rdparty/python/constraints.txt

```diff
@@ -11,14 +11,15 @@ fasteners==0.15
 idna==2.10
 importlib-metadata==2.0.0
 iniconfig==1.0.1
+libcst==0.3.12
```

Comment:
That's awesome that this is a pre-compiled wheel, and that it's so small (500 kb). I was concerned if this would have lots of transitive dependencies and/or be an sdist, because that would slow down the installation process for every Pants user.

Reply:
Yes! I think this is an awesome library! I've become one of those people who gets very wary about adding more 3rdparty deps into software we actually release -- it's been great to have that experience.